otlogginglibs/logging.py

```diff
@@ -0,0 +1,30 @@
+"""Library for logging."""
```

Comment:
Please add a README too with example to use this logger library.

Reply:
> Please add a README too with example to use this logger library.

  Added README with example and requirements file